mkat_fpga_tests/test_cbf.py

```diff
@@ -508,8 +555,7 @@ def get_expected_phases():
                 phases = self.corr_freqs.chan_freqs * 2 * np.pi * delay
                 phases -= np.max(phases)/2.
                 expected_phases.append(phases)
-
-            return zip(test_delays, expected_phases)
+            return zip(map(lambda x: round(x*1e9, 3), test_delays), expected_phases)
```

Comment:
Why are you rounding the expeecte phases? Surely you want the 'expected' value to be as accurate as possible and then just limit the precision when comparing? Otherwise you are introducing two sources of inaccuracy.